src/js/views/awardsRecords.js

```diff
@@ -120,7 +105,7 @@ function getPlayerAwards(p, awardType) {
     const formatYear = year => {
         return Object.keys(year).map(k => {
             const years = helpers.yearRanges(year[k].map(y => y.season)).join(', ');
-            return `${k} <small>(${years})</small>`;
+            return `${k} (${years})`;
```

Comment:
Change it to:

                return <span>{k} <small>({years})</small></span>;

and import React in this file.

However it would be better to keep things separated and not import React here, in which case you can assemble it in AwardsRecords.js instead of here, similar to what you did with the player name link. I think that is the better solution, although slightly more complicated. I think you'd have to move `formatYear` to AwardsRecords.js.